pkg/controller/kabaneroplatform/landing.go

```diff
@@ -392,6 +465,17 @@ func isInInterfaceList(urls []interface{}, s string) bool {
 
 // Retrieves the current kabanero landing page status.
 func getKabaneroLandingPageStatus(k *kabanerov1alpha1.Kabanero, c client.Client) (bool, error) {
+	// if landing page is disabled set landing status in kabanero to nil so it does not show up in kabanero's status
+	if k.Spec.Landing.Enable != nil && *(k.Spec.Landing.Enable) == false {
+		k.Status.Landing = nil
+		return true, nil
+	}
+	// landing page is optional. The status was defined as a pointer so that it is not displayed
+	// in the kabanero instance data if the landing page is disabled. That is because structures are
+	// never 'empty' for json tagging 'omitempty' to take effect.
```

Comment:
OK, so this explains why the landing page status is a pointer.  I'm not sure it's important that the `landing:` entry isn't present in the `Status` when the landing page is disabled.  I am concerned that this is going to cause trouble down the road by other components checking status and not realizing that the landing page status might be nil.  The same could be said for `Spec` but we should be the only ones setting that, and even if someone were programmatically creating a `Spec` for a Kabanero, `golang` would convert a `bool` to a `*bool` for them automatically on assignment.  If there is no other side effect other than not displaying the status when enabled is false, then I'd prefer keep this status as not a pointer and just have it be present but empty in the Status when the landing page is disabled.

Reply:
In the latest commit it is not a pointer.